cdap-api/src/main/java/co/cask/cdap/api/mapreduce/MapReduceContext.java

```diff
@@ -130,29 +130,6 @@
   void setInput(String datasetName, Dataset dataset);
 
   /**
-   * Overrides the output configuration of this MapReduce job to write to the specified dataset by its name.
-   *
-   * @param datasetName the name of the output dataset
-   * Deprecated as of version 3.2.0. Use {@link #addOutput(String)}, instead.
-   */
-  @Deprecated
-  void setOutput(String datasetName);
-
-  /**
-   * Overrides the output configuration of this MapReduce job to write to the specified dataset instance.
-   * Currently, the dataset passed in must be an {@link OutputFormatProvider}.
-   * You may want to use this method instead of {@link #setOutput(String)} if your output dataset uses runtime
-   * arguments set in your own program logic.
-   *
-   * @param datasetName the name of the output dataset
-   * @param dataset the output dataset
-   * @throws IllegalArgumentException if the given dataset is not an OutputFormatProvider.
-   * Deprecated as of version 3.2.0. Use {@link #addOutput(String, Map)}, instead.
-   */
-  @Deprecated
-  void setOutput(String datasetName, Dataset dataset);
-
-  /**
```

Comment:
I am not sure if this is a good idea. We still have setInput(). We should keep it consistent. Configure both input and output in the same way.